packages/synapse-client/out/modelGeneric.mustache

```diff
@@ -0,0 +1,168 @@
+import { mapValues } from '../runtime{{importFileExtension}}';
```

Comment:
This template file defines the generated code for a model defined in the OpenAPI spec. Most of this code is copied/generated from the original template source.

The lone change is denoted by `START CUSTOM TEMPLATING FOR synapse-client`